DFC.ServiceTaxonomy.PageLocation/ViewModels/PageLocationViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using OrchardCore.ContentManagement;
+using OrchardCore.DisplayManagement.Views;
+
+namespace DFC.ServiceTaxonomy.PageLocation.ViewModels
+{
+    public class PageLocationViewModel : ShapeViewModel
+    {
+        public PageLocationViewModel(ContentItem contentItem)
+        {
+            IsDefault = contentItem.ContentType == "Page" &&
+                Convert.ToBoolean(contentItem.Content.PageLocationPart.DefaultPageForLocation.Value);
```

Comment:
Would be good to use `nameof` for the part, and a public const string in the part for `DefaultPageForLocation`, to make this refactor friendly.